packages/react-vapor/src/components/tooltip/tests/DateTooltip.spec.tsx

```diff
@@ -17,16 +17,19 @@ describe('Tooltip', () => {
         });
 
         it('should display content in LLL if no format is given on moment date', () => {
+            defaultFormat = moment().format('LLL');
             tooltipWrapper = shallow(<DateTooltip date={moment()} />);
```

Comment:
I think it would be best to use a constant predefined moment instead of `moment()` which is equivalent to "now" and therefore always changes.

Reply:
Same logic applies for every other test

Reply:
I changed it for a constant so i can re-use it everywhere, poke me if that's not ok (@pochretien )